v2/tools/generator/internal/codegen/pipeline/apply_defaulter_and_validator_interfaces.go

```diff
@@ -181,6 +181,9 @@ func validateSecretDestinations(k *functions.ResourceFunction, codeGenerationCon
 		ReceiverIdent: receiverIdent,
 		ReceiverType:  astbuilder.PointerTo(receiverType),
 		Returns: []*dst.Field{
+			{
+				Type: astbuilder.QualifiedTypeName(astmodel.ControllerRuntimeAdmission.PackageName(), "Warnings"),
+			},
```

Comment:
It's easier and cleaner to use `fn.AddReturn()`, rather than trying to do it all in one statement.

Reply:
Done!